integreat_cms/cms/views/region_condition/region_condition_actions.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime
+from typing import TYPE_CHECKING
+
+import magic
+from django.db.models import Q
+from django.http import HttpResponse
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.http import require_POST
+from import_export import fields, resources
+from tablib.formats import registry as format_registry
+
+from ...constants import region_status, translation_status
+from ...decorators import permission_required
+from ...models import Page, Region
+from ...utils.linkcheck_utils import filter_urls
+from ..utils.hix import get_translation_under_hix_threshold
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.http import HttpRequest
+
+logger = logging.getLogger(__name__)
+
+
+class RegionConditionResource(resources.ModelResource):
+    """
+    A resources class that connects to the django-import-export library.
+    It represents the to-be exported status of all regions.
+    """
+
+    name = fields.Field(column_name=_("Region name"), attribute="name")
+
+    num_broken_links = fields.Field(column_name=_("Number of broken links"))
+
+    num_low_hix_pages = fields.Field(column_name=_("Number of low hix pages"))
+
+    num_pages_with_missing_or_outdated_translation = fields.Field(
+        column_name=_(
+            "Number of pages with at least one missing or outdated translation"
+        )
+    )
+
+    num_outdated_pages = fields.Field(column_name=_("Number of outdated pages"))
+
+    has_translation_package_been_booked = fields.Field(
+        column_name=_("Has translation package been booked"),
```

Comment:
I think adding a question mark makes it slightly better at conveying the meaning
```suggestion
    has_translation_package_been_booked = fields.Field(
        column_name=_("Has translation package been booked?"),
```